### ISSUE TITLE
Generating the documentation

### ISSUE THREAD

**Issue by PierreKiwi** [awsdocs/amazon-ec2-user-guide#101]
Hello,
I would like to generate the documentation myself (ideally in PDF) but the repository does not contain any script to do so.

Some have such as https://github.com/awsdocs/aws-ios-developer-guide
Feel free to close this issue if it is not relevant or supported.

Thanks!

**Comment by mariflax** [comment]
Hi,

You can get a PDF of the guide from our online docs: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html. In the content (middle) section, at top left, there's a "PDF" link in blue text. Choose the "PDF" link to open the guide as a PDF.

Is this what you were after?

Regards,
Marilyn Flax
AWS Documentation

**Comment by PierreKiwi** [comment]
Hello @mariflax,
Thanks for your quick reply.

Actually, I am not looking for the generated PDF but I would like to be able to generate the PDF myself.

Say I fork the current repository and I modify, I would like to be able to generate the PDF myself. I tried to adapt a script I found in some other repositories but no luck so far (the others repos are written in ReST but this one is written in Markdown).
